src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/repository/BookingRepository.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.repository
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.BookingEntity
+import java.util.UUID
+
+@Repository
```

Comment:
I've started defining the repository in the same file as the entity - they are very much coupled anyway and the first thing I always do with a Repository is click on the Entity!

Reply:
+1 to this idea.

Reply:
This was addressed in https://github.com/ministryofjustice/approved-premises-api/pull/30